src/Cards/Cards.js

```diff
@@ -30,14 +32,36 @@ export default class Cards extends Component {
     });
   }
 
+  sortMovies = (value) => {
+    if (!value || value === '--none--') {
+      return false
+    } else if (value === 'descending'){
+      const sortedMovies = this.state.movies.sort((a, b) => a.average_rating - b.average_rating)
+      this.setState({ movies: sortedMovies })
+    } else if (value === 'ascending'){
+      const sortedMovies = this.state.movies.sort((a, b) => b.average_rating - a.average_rating)
+      this.setState({ movies: sortedMovies })
+    } 
```

Comment:
I want to make use of the `value` parameter here instead of writing in the `a.average_rating - b.average_rating` but for some reason, it is not working. If I write `a.value - b.value` it odes not do the sorting. If you can give an extra review on this @holladayian  I will really appreciate it! thanks.

Reply:
Hmm, so I think because none of the movies from `this.state.movies` has a `.value`, we can't sort by value. I know writing `b.average_rating - a.average_rating` looks kind of clunky, but this appears to be the correct way.

Reply:
Yeah it'll be dependent on what the key names on those movie objects are.